frontend/app-development/features/appPublish/components/DeploymentEnvironmentStatus.tsx

```diff
@@ -82,25 +83,29 @@ export const DeploymentEnvironmentStatus = ({
     <DeploymentStatusAlert
       severity='success'
       content={
-        <Trans
-          i18nKey={'app_deployment.status.succeeded'}
-          values={{
-            version: kubernetesDeployment.version,
-          }}
-          components={{
-            a: <Link href={urlToApp}> </Link>,
-          }}
-        />
-      }
-      footer={
-        lastPublishedDate && (
+        <>
           <Trans
-            i18nKey={'app_deployment.last_published'}
+            i18nKey={'app_deployment.status.succeeded'}
             values={{
-              lastPublishedDate: formatDateTime(lastPublishedDate),
+              version: kubernetesDeployment.version,
+            }}
+            components={{
+              a: (
+                <Link href={urlToApp} rel='noopener noreferrer' target='_blank'>
+                  {' '}
+                </Link>
+              ),
+              ext: <ExternalLinkIcon />,
```

Comment:
I think we can use the `title` of the svg to make it visible only for screen readers

```suggestion
              ext: <ExternalLinkIcon title={t('app_deployment.status.open_app_in_new_window')} />,
```